templates/dhcpd.pool.erb

```diff
@@ -20,14 +20,14 @@ subnet <%= @network %> netmask <%= @mask %> {
 <%   @options.each do |opt| -%>
   option <%= opt %>;
 <%   end -%>
-<% elsif @options != '' -%>
+<% elsif @options and !@options.empty? -%>
```

Comment:
Nitpick, in core we'd say you should use `&&` in an expression rather than `and`.  I don't recall if we enforce that in the installer or not.